### ISSUE TITLE
Provide an easier way to set aria-label

### ISSUE THREAD

**Issue by silvangrueter** [zHaytam/SmartBreadcrumbs#59]
In order to be compliant with the WCAG guidelines, one should set a aria-label to the breadcrumb navigation, if there are more than one <nav> element on the page. This is the case on most pages.

There is a workaround by appying the aria-label to the TagName of the BreadcrumbOptions like this:

`o.TagName = "nav aria-label=\"Some description of the navigation\"";`

An appropriate property would be a better way to handle this and should be fairly easy to implement.

**Comment by zHaytam** [comment]
I agree, there should be a way to do it that doesn't feel like a hack.  
I'll try to find the time for it.

**Comment by zHaytam** [comment]
Added in 3.4!
